### DISCUSSION TITLE
What issues are you currently facing with the plugin, or what updates would you like to see?

### DISCUSSION THREAD

**Discussion by eslam-dev** [developvi/dvi-cloud-deploy#3] [General]
We are committed to improving the plugin and making it more efficient and stable. We are interested in understanding the issues you are currently facing while using the plugin, as well as any updates or features you think are necessary or beneficial to include in future releases. Please provide a detailed description of the issues or suggestions you have so we can address them effectively.

**Comment by qvdrei** [comment]
@eslam-dev sure thing! If we are to seriously develop this project, we ne a feature request board, similar to what nigel already had.

https://trello.com/b/pYl53rvM/wpcloud-deploy-public-roadmap

You can already see what Nigel was working on.

Also, we can use a paid solution (which I recommend), https://www.feedbear.com/, but only when we have more users.

**Reply by eslam-dev** [reply]
@qvdrei 
Thank you very much! I will start working on the ideas available in Trello.

If you have any additional suggestions, please feel free to share them.

**Comment by MaqAnquor** [comment]


**Reply by eslam-dev** [reply]
@nightcrawler123 
Yes, we are currently working on supporting Laravel in WPCD.

**Reply by MaqAnquor** [reply]
Have you guys created a timeline for release?

**Reply by eslam-dev** [reply]
@nightcrawler123 
We are working on developing and testing the plugin, but we have not set a release date yet.

**Reply by MaqAnquor** [reply]
If you want me to test it out reach out to me.

**Reply by eslam-dev** [reply]
@nightcrawler123 
You can try the new version that was uploaded yesterday and let me know if you have any suggestions. A version that supports Laravel will be uploaded soon.

**Reply by MaqAnquor** [reply]
https://discord.com/invite/eWv5KX3Q

Hey, can you join the discord server!

**Reply by eslam-dev** [reply]
@nightcrawler123 
I have indeed joined the Discord group, but it seems to me that there is nothing to talk about or any ideas to develop the plugin with.

**Reply by MaqAnquor** [reply]
Oh, please react on the rules page and the other options would appear.

"React to the rules to gain access to the rest of the server 🙂"![Screenshot_2024-08-01-14-15-42-22_572064f74bd5f9fa804b05334aa4f912.jpg](https://github.com/user-attachments/assets/0399aaa4-5f2f-4a72-9788-11f7c81c367f)

**Reply by qvdrei** [reply]
> @nightcrawler123 I have indeed joined the Discord group, but it seems to me that there is nothing to talk about or any ideas to develop the plugin with.

I will provide some feature requests in the discord.

**Comment by MaqAnquor** [comment]
ability to add custom ssl using GUI

**Reply by eslam-dev** [reply]
@nightcrawler123 
We will add the ability to add custom SSL using the GUI to the development list.

**Comment by gkantsios** [comment]
Hi @eslam-dev!  We're in the process of setting up WPCD for the first time and need some assistance. We tried to join the Discord group, but that invite isn't valid.

If we're interested in the future development + possibly hiring someone to help, can you do so?

**Reply by qvdrei** [reply]
> Hi @eslam-dev! We're in the process of setting up WPCD for the first time and need some assistance. We tried to join the Discord group, but that invite isn't valid.
> 
> If we're interested in the future development + possibly hiring someone to help, can you do so?

Here's the invite link:

https://discord.com/invite/rRt9ezM8

**Reply by gkantsios** [reply]
@qvdrei thank you!
